node/node_genesis.go

```diff
@@ -47,14 +50,12 @@ func (gi *genesisInitializer) InitChainDB(db ethdb.Database, shardID uint32) err
 		}
 		shardState = types.ShardState{*c}
 	}
-	if err := gi.node.SetupGenesisBlock(db, shardID, shardState); err != nil {
-		return ctxerror.New("cannot setup genesis block").WithCause(err)
-	}
+	gi.node.SetupGenesisBlock(db, shardID, shardState)
```

Comment:
why error is ignored?

Reply:
I switched to MustCommit which will panic.